Moderation-Policy.md

```diff
@@ -75,6 +76,26 @@ subject to Moderation.
 The Moderation Team is responsible for deciding what constitutes inappropriate
 behavior that may be subject to Moderation.
 
+## Outside Commentary via Social Media
+
+Comments by individuals made in any venue falling outside the explicit scope of
+responsibility of the Node.js Foundation, TSC, or Community Committee are
+explicitly not covered by this policy unless the individual is explicitly acting
+or commenting on behalf of the Node.js Foundation, TSC or Community Committee
+in an official capacity. The statements, views, and opinions contained in such
+comments are not endorsed by, nor do they necessarily reflect, the opinions and
+views of the Node.js Foundation, TSC or Community Committee.
+
+Moderation requests based solely on the content of comments made in any such
+venue *may* be ignored.
```

Comment:
I have the strong feeling that this will only exacerbate the current situation. 

The primary critique seemed to be:
- Why does `person A`'s twitter post violate the CoC, but not `person B`'s inflammatory twitter post?

@jasnell at the end of the day: non-github, personal social media posts need to be either `in-scope` or `out-of-scope` for CoC enforcement.

Reply:
Yes, I absolutely get this. I have a distinct feeling that *most* people would like to be able to give a blanket statement that says what you post to your own twitter account is your own business, but unfortunately that is just not realistic because people will associate the comments with the entire project *no matter what our policy says*. So we need to find the right balance and ensure that it is applied equally across the board. This is just a first draft attempt at getting to that.

The underlying idea here *should* be simple. Basically it is: Posts to social media outside the Node.js project are outside the scope of this policy *unless they obviously aren't*.... Except, of course, defining that *unless they obviously aren't* part that becomes the most difficult. What is the right threshold? What would the community consider to be the right threshold.

Reply:
Case in point. If we say that posts to social media are out of scope, then immediately jumped on twitter making high derogatory claims against specific fellow collaborators, would I be in violation of the code of conduct or not? By the letter of the law, sure, but definitely not by the spirit. We have to do better than that.

Reply:
> If we say that posts to social media are out of scope, then I immediately jumped on twitter making high derogatory claims against specific fellow collaborators because hey, those don't count, would I be in violation of the code of conduct or not? 

People on twitter making unfounded claims is what started the twitter firestorm with Rod. **I support a collaborators right to have a negative opinion about me.** What do you think someone could say that wouldn't be against Twitters ToS?

The problem starts when derogatory claims make their way into a professional space. **That's the part you can control.**

> because hey, those don't count, would I be in violation of the code of conduct or not? 

In a perfect world: 
- No it should not be. 
- Block them and move on.

If you do this, people will just make throwaways to post such things...It already happened yesterday: https://www.reddit.com/user/node_in_peace

Reply:
@jakeNiemiec you and @mscdex make similar points (https://github.com/nodejs/TSC/pull/327#discussion_r135913402) so I'll address them both at once here: Yes, you're both absolutely correct. Giving the "out" here and allowing it to be case by case *does* open a loophole. So does *not* giving the out, unfortunately.

@jakeNiemiec ... your comment "That's the part you can control" is absolutely spot on. Unfortunately, it's not the comments made within the arena we can control that cause the most problems, it's nearly always the comments that are made on Twitter, or Reddit, etc. Those are the channels that most Node.js users pay attention to and have the most visibility on. More people pay more attention to my personal twitter feed than they do my comments in the issue tracker.

But then, I know, and I think everyone understands and accepts, then whenever I post something about the current political situation in the United States to my personal twitter that I am expressing my own personal opinion. There's just something fundamentally *different* about such posts, just as there is something fundamentally different about the various `PSA: Something happened in Node.js` posts I make to the exact same channel. They are rightfully perceived differently. The posts have different intent and goals. The PSA posts I make cannot be wholly separated from me acting as a member of the core project. And *that* is where the complication lies.

Back when I worked at IBM, a long long time ago I helped write the very first version of the company social media guidelines for employees. The most difficult thing we went back and forth on was the idea about whether an employee tweeting on their personal account represented the company or not, and we landed on it just being a giant grey area. Whatever you say *will* be associated with whatever organization you are associated with. Period. In a perfect world, that's not fair, but it *is* the reality. Another reality is that people do not give their words enough consideration or measure.